app/controllers/forms_controller.rb

```diff
@@ -35,19 +35,21 @@ def update
     flash[:message] = "Successfully updated!"
     redirect_to action: "show", id: form.id
   rescue StandardError
-    flash[:message] = "Unsuccessful"
+    flash[:message] = "Update unsuccessful"
     redirect_to :edit_form, id: params[:id]
   end
 
   def destroy
     form = Form.find(params[:id])
-    form.destroy!
-
-    flash[:message] = "Successfully deleted #{form.name}"
-    redirect_to root_path, status: :see_other
+    if form.destroy
```

Comment:
For some reason `form.destroy!` throws an undefined method error, and Rubocop doesn't like form.destroy without the if statement, which is why I've gone for this solution

Reply:
Lol ok then - I would say it's cos ActiveResource probably doesn't define a `destroy!` method 🤔 which is because it does them all loud by default, but then rubocop will complain yeah cos it expects it to return true/false. Oof - we could disable that linting if it turns up again in the future with the reason that active resource doesn't work like that I guess